politeiawww/politeiawww.go

```diff
@@ -845,7 +847,7 @@ func (p *politeiawww) handleWebsocketRead(wc *wsContext) {
 }
 
 // handleWebsocketWrite attempts to notify a subscribed websocket. Currently
-// only ping is supported.
+// only ping & quiesce are supported.
```

Comment:
We're going to be adding more websocket events so you can just take this sentence out.  It will not longer apply.